readme.md

```diff
@@ -184,6 +184,7 @@ Made with Electron.
 - [Etcher](https://github.com/resin-io/etcher) - Flash OS images to SD cards and USB drives.
 - [Noty](https://github.com/fabiospampinato/noty) - Auto-saving sticky note with support for multiple notes in a single window.
 - [Notable](https://github.com/fabiospampinato/notable) - Markdown-based note-taking.
+- [Unsplash Wallpapers](https://github.com/soroushchehresa/unsplash-wallpapers) - Unsplash cross-platform desktop application.
```

Comment:
Needs a better description. It should succinctly describe what the app does.

Reply:
@sindresorhus I updated the description.